tests/testlogout.js

```diff
@@ -0,0 +1,31 @@
+import LoginPage from '../pages/login.page';
+import HomePage from '../pages/home.page';
+import LogoutPage from '../pages/logout.page';
+//import { expect } from 'chai';
+
+describe ('TestLogout', async () => {
+
+    it('Should login and logout successfully', async () => {
+
+        await LoginPage.open('login');
+        await LoginPage.loginAs('jsmith@demo.io', 'Demo123!');
+        await expect(HomePage.welcomeUser).toHaveTextContaining('Welcome Josh');
```

Comment:
This is not really needed for this test, as it's basically asserting that the login happened successfully, which is not the goal of this test (assertions need help validate the main goal of the test case)